components/brave_ads/browser/ads_service_impl.h

```diff
@@ -71,8 +74,10 @@ class AdsServiceImpl : public AdsService,
   void OnMediaStop(SessionID tab_id) override;
   void ClassifyPage(const std::string& url, const std::string& page) override;
   void SetConfirmationsIsReady(const bool is_ready) override;
+  bool IsTestingEnv() const override;
 
   void Shutdown() override;
+  void MaybeShowFirstLaunchNotification() override;
```

Comment:
Agreed with you this is not needed here, please remove

Reply:
We don't need it in `ads_service.h` but it's still a member of `AdsServiceImpl` so I'll just move this to private.